lib/mongo/retryable.rb

```diff
@@ -488,24 +63,24 @@ def select_server(cluster, server_selector, session)
       end
     end
 
-    # Log a warning so that any application slow down is immediately obvious.
-    def log_retry(e, options = nil)
-      message = if options && options[:message]
-        options[:message]
-      else
-        "Retry"
-      end
-      Logger.logger.warn "[jontest] #{message} due to: #{e.class.name}: #{e.message}"
```

Comment:
The `[jontest] ` prefix is removed in this merge, and I don't think we want it to be (in case it is in Datadog log monitors, etc.)

it looks like the method is now defined in `lib/mongo/retryable/base_worker.rb`